src/ember-app.js

```diff
@@ -37,10 +37,15 @@ class EmberApp {
     this.vendorFilePaths = config.vendorFiles;
     this.moduleWhitelist = config.moduleWhitelist;
     this.hostWhitelist = config.hostWhitelist;
-    this.appConfig = config.appConfig;
+    this.config = config.config;
+    this.appName = config.appName;
 
     if (process.env.APP_CONFIG) {
-      this.appConfig = JSON.parse(process.env.APP_CONFIG);
+      let appConfig = JSON.parse(process.env.APP_CONFIG);
+      let appConfigKey = `${this.appName}/config/environment`;
```

Comment:
Should the keys be `${appName}/config/environment` or just `appName`?  I'd generally prefer the config keys to be "opaque" (so either is fine), but I'd lean towards making the key in this object just `appName` here (though I don't feel _super_ strongly).

Reply:
I am fine with anything. The reason I went with `appName/config/environment` since it is easier to map to the AMD module name.